app/javascript/src/hyacinth_v1/components/projects/publish_targets/PublishTargetsForm.js

```diff
@@ -0,0 +1,158 @@
+import React, { useEffect, useState } from 'react';
+import { useMutation, useQuery } from '@apollo/react-hooks';
+import PropTypes from 'prop-types';
+import {
+  Row, Col, Form, Badge,
+} from 'react-bootstrap';
+import { useHistory } from 'react-router-dom';
+import produce from 'immer';
+import GraphQLErrors from '../../shared/GraphQLErrors';
+import FormButtons from '../../shared/forms/FormButtons';
+import { getProjectsPublishTargetsQuery, updateProjectsPublishTargetsMutation } from '../../../graphql/projects/publishTargets';
+
+const PublishTarget = (props) => {
```

Comment:
```suggestion
const AvailablePublishTarget = (props) => {
```

This suggestion is related to my earlier renaming suggestion.

Reply:
Done!